ModList/UI/ViewControllers/ModControlsViewController.cs

```diff
@@ -1,21 +1,38 @@
-﻿using BeatSaberMarkupLanguage.Attributes;
-using BeatSaberMarkupLanguage.Parser;
-using BeatSaberMarkupLanguage.ViewControllers;
-using IPA.Loader;
-using System;
+﻿using System;
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using BeatSaberMarkupLanguage.Attributes;
+using BeatSaberMarkupLanguage.Parser;
+using BeatSaberMarkupLanguage.ViewControllers;
+using IPA.Loader;
+using IPA.ModList.BeatSaber.Models;
+using IPA.ModList.BeatSaber.Services;
+using ModestTree;
+using SiraUtil.Tools;
+using Zenject;
 
-namespace IPA.ModList.BeatSaber.UI
+namespace IPA.ModList.BeatSaber.UI.ViewControllers
 {
-    [HotReload(PathMap = new[] { "C:\\", CompileConstants.SolutionDirectory })]
-    public class ModControlsViewController : BSMLAutomaticViewController
+    [HotReload(RelativePathToLayout = @"..\Views\ModControlsView.bsml")]
+    [ViewDefinition("IPA.ModList.BeatSaber.UI.Views.ModControlsView.bsml")]
```

Comment:
I generally prefer having the BSML next to the ViewController exactly so that this attribute isn't necessary, as well as generally liking those things to go together since their inextricably tied to each other anyway.